src/grammar.lalrpop

```diff
@@ -1094,7 +1094,7 @@ UpName: String = {
 }
 
 DiscardName: String = {
-    <v:r"_[A-Za-z_]*"> => v.to_string()
+    <v:r"_[a-z][0-9a-z_]*"> => v.to_string()
```

Comment:
Does this regex work for `_` alone not followed by anything? Maybe it should be this way or something like that:
```suggestion
    <v:r"_[a-z]*[0-9a-z_]*"> => v.to_string()
```

Reply:
I think that one would allow `_0`, which I think we want to avoid.

Maybe this?

```
<v:r"_([a-z][0-9a-z_]*)*"> => v.to_string()
```

Tested here -> https://rustexp.lpil.uk/

Reply:
I thought `_([a-z][0-9a-z_]*)?` might be simpler.

Neither seems to work with a discard pattern of just `_` (on your rustexp page)

Reply:
Ah yes! That's much better